test/test_encryption.py

```diff
@@ -1105,5 +1107,141 @@ def test_05_endpoint_invalid_host(self):
                 'aws', master_key=master_key)
 
 
+class AzureGCPEncryptionTestMixin(object):
+    KMS_PROVIDER_MAP = None
+    KEYVAULT_DB = 'keyvault'
+    KEYVAULT_COLL = 'datakeys'
+
+    def read_extjson(self, filename):
+        with open(os.path.join(BASE, 'custom', filename), 'r') as fp:
+            return json_util.loads(fp.read(), json_options=JSON_OPTS)
+
+    def setup_keyvault(self, DEK):
+        keyvault = self.client.get_database(self.KEYVAULT_DB).get_collection(
+            self.KEYVAULT_COLL, codec_options=OPTS,
+            write_concern=WriteConcern('majority'))
+        keyvault.drop()
+        keyvault.insert_one(DEK)
+
+    def _test_explicit(self, expectation):
+        client_encryption = ClientEncryption(
+            self.KMS_PROVIDER_MAP,
+            '.'.join([self.KEYVAULT_DB, self.KEYVAULT_COLL]),
```

Comment:
Suggest refactoring `'.'.join([self.KEYVAULT_DB, self.KEYVAULT_COLL])` into a variable to make this more readable.